modules/user-access/pages/user-management.adoc

```diff
@@ -9,10 +9,36 @@ To see user management tasks under the xref:access-control-model.adoc#_access_co
 
 You can run the `CREATE USER` command to create a user.
 
-The username cannot contain the following characters: `\` ,` ` `(` , `)`, `[`, `]`, `:`, `<`, `>`, `;`, `,`, `@`, `\r`, `\n`, `\f`, `\t`, `\\`, `\0`, `\b`.
-It also cannot start with a dot `.` or have multiple dots in a sequence.
+== Username Rules for v3.9.3 and above
 
-You can use non-ascii characters, such as Chinese and Kanji characters.
+NOTE: v3.9.3 expanded the variety of characters users can access while creating a username from previous versions.
+
+IMPORTANT: A username is treated as a `STRING` in GSQL,
+regardless of the syntax structure, format, or characters used.
+
+*Usernames* may contain any Unicode characters *except* the following:
+
+. May not contain these characters: `\r`, `\n`, `\0`, `\b`, `\t`.
+
+. May not contain any `SPACE`.
+
+. May not contain a colon ( `:` ).
+
+NOTE: Backquotes ( ``` ) must be used as a delimiter to enclose a username which does not follow
+the pattern `[a-zA-Z_][a-zA-Z0-9]*`. Delimiters are not stored internally.
```

Comment:
NOTE: Backquotes ( ``` ) must be used as a delimiter to enclose a username which _is not a valid email and_ does not follow
the pattern `[a-zA-Z_][a-zA-Z0-9]*`. Delimiters are not stored internally.

Actually, it is also optional when createing user since it is a sperate dialog with user.